src/channel.rs

```diff
@@ -85,6 +87,7 @@ fn dur_to_ms(dur: Duration) -> i32 {
 enum Options {
     Integer(i32),
     String(CString),
+    Pointer(*mut c_void, *mut c_void),
```

Comment:
Memory is leak if options is not used.

Reply:
Can you explain in detail?

Reply:
For example, if the builder is created and configured, but never call `build_args` or `connect`, and then dropped.

Reply:
I think it might be good to have a struct called something like PointerVtable with named fields (having two `*mut c_void` pointers scares me) and then you can call the destroy function in the destructor.

Reply:
@BusyJay @nrc 
Done! `ChannelBuilder` is responsible for destroying the pointer of its own reference when destroying.